### DISCUSSION TITLE
Whitelisting Wildcard Domains

### DISCUSSION THREAD

**Discussion by r1bnc** [AdguardTeam/AdGuardHome#3548] [Q&A] [answered]
Hello, I am trying to whitelist wildcard domain for some reason a legitimate site is being blocked by the Parental Control. I tried to add this line on the `Custom Filtering Rules` but it doesn't seem to work:

```
@@||domain.com^$important
```
From what I understand this should unblock *.domain.com. But the Parental Control still overrides it. Perhaps I made a mistake.

**Comment by ainar-g** [comment, marked as answer]
Hello.  We cannot reproduce the issue.  Are you sure that you spelled the domain name correctly in your custom rule?

**Reply by r1bnc** [reply]
Seems to work now. Will check again if not resolved. Thank you.
